extensions/query-store/src/common/utils.ts

```diff
@@ -75,7 +75,11 @@ export function createSplitView(view: azdata.ModelView, firstComponent: azdata.C
 	return splitview;
 }
 
-export enum ConfigInfo {
-	overallResourceConfig,
-	topResourceConfig
+export enum ConfigComponentsInfo {
```

Comment:
add doc comment that these are the available components that can be added to a config dialog